views/js/runner/plugins/controls/timer/strategy/guidedNavigation.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * Timer strategy that enforce the test taker to stay in
+ * front of the item until the timer completes,
+ * by hidding the navigation elements.
+ * Once the timer completes, the test moves to the next item.
+ *
+ * Applies on item scope, locked timers only, linear test part and
+ * the test configuration "guidedNavigation"
+ *
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([], function(){
+    'use strict';
+
+    /**
+     * Creates the strategy if it applies to the given timer
+     * @param {runner} testRunner
+     * @param {Object} timer
+     * @returns {strategy|Boolean} the strategy if applies or false
+     */
+    return function guidedNavigationStrategy(testRunner, timer){
+        var testData = testRunner.getTestData();
+        var testContext = testRunner.getTestContext();
+        var pluginConfig = testData && testData.config && testData.config.plugins && testData.config.plugins.timer;
+
+        if( timer && timer.type === 'locked' && timer.scope === 'item' &&
```

Comment:
No. The agreement has remained on this strategy "min/max" + platform option. 
Having the `guidedNavigation` as a standalone strategy will enable us to activate it with whatever you want as a condition with a minimal effort (ie. adding `|| (type === 'max' && yetAnotherCustomCategory)`. Without changing the other behaviors, so the goal of strategy is decoupling the behavior so we can easily activate/deactivate them).

Reply:
> Other concern if I set min === max in authoring without locking, and with the guidedNAvigation option enabled, then this will not work. Which means that basically the lock button behaves like a "enable guided Navigation" button, and this just does not feels right @markmolenaar / @ssipasseuth

Reply:
indeed, my bad!